cmd/kubeadm/app/apis/kubeadm/env.go

```diff
@@ -34,6 +34,7 @@ func GetEnvParams() map[string]string {
 		"host_pki_path":      "/etc/kubernetes/pki",
 		"host_etcd_path":     "/var/lib/etcd",
 		"hyperkube_image":    "",
+		"repo_prefix":	      "",
```

Comment:
I think it would be better to set this to `gcr.io/google_containers` by default, maybe via the gcrPrefix constant